samples/pet/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13.3)
+project (sample_pet)
+
+# Generate encoding and decoding code for the 'Pet' structure.
+set(py_command_pet
+  zcbor code -d -e -b64 --short-names
+  -c ${CMAKE_CURRENT_LIST_DIR}/../../tests/cases/pet.cddl
+  -t Pet
+  --output-cmake ${PROJECT_BINARY_DIR}/pet.cmake
+  )
+execute_process(
+  COMMAND ${py_command_pet}
+  COMMAND_ERROR_IS_FATAL ANY
+  )
+
+# Convert the data in the pet1.yml file to CBOR and put the data in a header
+# file as a C array.
+set(py_command_convert_pet1
+  zcbor convert -c ${CMAKE_CURRENT_LIST_DIR}/../../tests/cases/pet.cddl -t Pet
+  -i ${CMAKE_CURRENT_LIST_DIR}/pet1.yml
+  -o ${PROJECT_BINARY_DIR}/include/pet1.h --c-code-var-name pet1
+  )
+execute_process(
```

Comment:
Do we really need to use `execute_process` here? Couldn't it be possible to make the `app` executable depend on some target which is used to execute these commands? Again since this is example code we should display best practices (and I assume that we want to avoid `execute_process` if possible.